capstone-backend/controllers/dailyCalorie.controller.js

```diff
@@ -0,0 +1,39 @@
+const DailyCalorie = require("../models/dailyCalorie.model");
+const dotenv = require("dotenv");
+dotenv.config("../.env");
+
+const addDailyCalorie = async (req, res) => {
+  try {
+    // let currentDate = new Date();
+    console.log(Date());
+    const obj = {
+      Date: Date(),
```

Comment:
This date will be obtained from front-end.
Also add a validation to check if the date is <= today's date

Reply:
Done